### PR TITLE
Adding the ability to remove the menu bar altogether by passing "null" to toolbar prop of menu.

### PR DESCRIPTION
**Pull request by ARKEOLOGIST**
# I'm submitting a...

- [ ] Bug Fix
- [X] Feature
- [ ] Other (Refactoring, Added tests, Documentation, ...)

### Checklist

- [X] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
  - A document related commit is prefixed "docs:"
- [ ] Docs have been added / updated (for bug fixes / features)

### Description

Just added the option to remove the toolbar altogether by setting the toolbar prop to "null".

### Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### Does this PR affects any existing issues?

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by sibiraj-s** [comment]
@ARKEOLOGIST thanks for the PR, can you remove the style related changes. You can run `ng lint --fix` to automatically fix those.

File: src/plugins/menu/menu.ts [hunk 1]
```diff
@@ -298,70 +298,72 @@ export const renderMenu = (options: MenuOptions, editorView: EditorView, menuDom
...
-          if (DROPDOWN_ITEMS.has(dropdownGroup)) {
-            const dropdown: ToolbarDropdownGroupValues = toolbarItem[dropdownGroup];
-
-            const dropdownView = new DropDownView(dropdownGroup, dropdown, editorView, options);
-            const rendered = dropdownView.render();
-            updates.push(rendered.updates);
-            menuDom.appendChild(rendered.dom);
-          } else {
-            console.warn('Unkown dropdown group:', dropdownGroup);
+  if (toolbar !== null) //handling excess div space rendered if toolbar is not to be rendered (at all).
```
Comment (sibiraj-s) [inline src/plugins/menu/menu.ts:301]: you could just return when it is null or undefined. so that changes will be minimal.

**Review by sibiraj-s — CHANGES_REQUESTED** [review]
Once you address the comments, Ping me. I will review and merge.

**Comment by ARKEOLOGIST** [comment]
Hi,

I have made the listing changes. 
I was wrong about including the "if" condition there. It hides the text editor and causes problems.

The solution is not to insert the DOM node at all if the option sent is null. Made the necessary changes.

File: src/plugins/menu/menu.ts (hunk 1 above)
Comment (ARKEOLOGIST) [reply src/plugins/menu/menu.ts]: Hi, I have made changes addressing this.

**Comment by sibiraj-s** [comment]
@ARKEOLOGIST You have to remove all the changes, even styling from this PR if this is not related to the fix provided.

**Comment by ARKEOLOGIST** [comment]
Hi, 

All changes are done by prettier it seems. The only changes I made are in MenuView.ts.

Do I revert the changes to these documents? 

Sorry for the bad questions.

**Pull request closed without merging**